apps/elewa-website/src/app/app.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component } from '@angular/core';
+import { ContentText } from 'libs/models/schema/ui/texts/content-text.interface';
 
```

Comment:
You should use aliased imports(`@elewa...`). The reason for the error above is because your model/schema is not a module that exports the interface contentText